dicom-archive/get_dicom_info.pl

```diff
@@ -136,7 +136,8 @@ =head2 Methods
     my(@position) = 
 	# ImagePositionPatient (0x0020, 0x0032)
         &convert_coordinates(&split_dicom_list(&trim($dicom->value('0020', '0032'))));
-    if (scalar(@position) != 3) {
+    my $computeSlicePos = grep($_->[1] eq 'slicepos', @Variables);
+    if (scalar(@position) != 3 && $computeSlicePos) {
        warn "Warning: The file: $filename is not DICOM!\n";
```

Comment:
@nicolasbrossard I wonder if we should not update the warning error to something clearer. Like "Warning: the file $filename is either not DICOM or does not contain any value for DICOM header (0020,0032)"

Reply:
@cmadjar Since header (0020,0032) is not mandatory, I vote for changing the warning to

 "DICOM header (0020,0032) not found in $filename: slice position cannot be computed. Skipping file.\n"

What do you think?

Reply:
Perfect!!!